packages/tpetra/core/src/Tpetra_BlockMultiVector_decl.hpp

```diff
@@ -474,26 +474,25 @@ class BlockMultiVector :
     mv_.sync_device();
   }
 
-#endif // TPETRA_ENABLE_DEPRECATED_CODE
-
   //! Whether this object needs synchronization to the given memory space.
   template<class TargetMemorySpace>
+  //TPETRA_DEPRECATED
```

Comment:
Is this commenting out intentional, to avoid spamming app developers with warnings?

Reply:
that's correct.  Geoff did it already for the rest of the things we deprecated on Friday, I believe